docs/editor/intellisense.md

```diff
@@ -0,0 +1,162 @@
+---
+Order: 4
+Area: editor
+TOCTitle: IntelliSense
+ContentId: 80f4fa1e-d4c5-42cf-8b12-4b8e88c41c3e
+PageTitle: IntelliSense in Visual Studio Code
+DateApproved: 8/15/2016
+MetaDescription: One of the core features of Visual Studio Code is IntelliSense.  Set breakpoints, step-in, inspect variables and more.
+---
+
+# IntelliSense
+
+IntelliSense is a general term for a variety of code editing features, including: code completion, parameter info, quick info, and list members. IntelliSense is described in a number of ways such as "code completion", "content assist", and "code hinting."
+
+![IntelliSense demo](images/intellisense/intellisense.gif)
+
+## Where is IntelliSense for my language? 
+
+IntelliSense is provided in VS Code for JavaScript, TypeScript, JSON, HTML, CSS, Less, and Sass out of the box. Other languages have word completions and can be configured to have IntelliSense by installing a language extension. 
+
```

Comment:
As mentioned below, add something to describe the difference between word-based completions and intelligent completions provided by a language service and how these are provided by language extensions.